.github/workflows/fbgemmci.yml

```diff
@@ -68,3 +68,40 @@ jobs:
         cd build_shared
         ctest
 
+  build1:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+
+    - name: Build static FBGEMM lib
+      shell: cmd
+      run: |
+        set
+        echo %PATH%
+        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
+        echo "INSTALL NINJA:"
+        pip install ninja
+        which ninja
+        mkdir build_static
+        cd build_static
+        echo "STARTING CMAKE"
+        cmake -G Ninja -DFBGEMM_BUILD_BENCHMARKS=OFF -DFBGEMM_BUILD_TESTS=OFF -DFBGEMM_LIBRARY_TYPE=static -DCMAKE_C_COMPILER="cl.exe" -DCMAKE_CXX_COMPILER="cl.exe" ..
+        ninja all
+        echo "Build Success"
```

Comment:
Does it only echo Build Success if the previous command succeeds?

Reply:
I think so since whenever error happens, ninja will stop the whole process.

Reply:
Wait, isn't this cmd? For commands running in CMD, it won't stop on error. You may have to check manually with something like `if errorlevel 1 exit /b 1`.

Reply:
I will made the changes. Thx.